tests/Context/RuntimeContextTest.php

```diff
@@ -53,23 +53,23 @@ public function valuesDataProvider(): array
                 ],
                 [],
                 UndefinedOptionsException::class,
-                'The option "foo" does not exist. Defined options are: "name", "version".',
+                '/The option "foo" does not exist. Defined options are: "name", "version"./',
```

Comment:
As this is a regex now, you should escape all special characters like the dot. Also, please use `^` and `$` to make sure that the matching is done against the string as a whole